ci-operator/step-registry/cucushift/installer/rehearse/azure/upi/baselinecaps/v411/provision/cucushift-installer-rehearse-azure-upi-baselinecaps-v411-provision-chain.yaml

```diff
@@ -0,0 +1,19 @@
+chain:
+  as: cucushift-installer-rehearse-azure-upi-baselinecaps-v411-provision
+  steps:
+  - ref: ipi-install-rbac
+  - chain: ipi-conf-azure
+  - ref: upi-install-azure
+  - ref: enable-qe-catalogsource
+  env:
+    - name: BASELINE_CAPABILITY_SET
+      default: "v4.11"
```

Comment:
Will this workflow only works with 4.11?

Reply:
There is no explicit limitation that v4.11 is only used in ocp4.11, since the content of v4.12 does not come out, maybe we can let this workflow run in 4.11 now. If any change in the future, cover that in installer test case.

Reply:
Based on current implements on 4.12, v4.11 is still supported in release 4.12, so I think this workflow should be run on 4.11+, updated the description of this chain.